operator/v1/types_network.go

```diff
@@ -788,3 +814,38 @@ const (
 	// between pods on the cluster network.
 	IPsecModeFull IPsecMode = "Full"
 )
+
+// +kubebuilder:validation:Enum:="";"Enabled";"Disabled"
+type RouteAdvertisementsEnablement string
+
+var (
+	// RouteAdvertisementsEnabled enables route advertisements for ovn-kubernetes
+	RouteAdvertisementsEnabled RouteAdvertisementsEnablement = "Enabled"
+	// RouteAdvertisementsDisabled disables route advertisements for ovn-kubernetes
+	RouteAdvertisementsDisabled RouteAdvertisementsEnablement = "Disabled"
+)
+
+// RoutingCapabilitiesProvider is a component providing routing capabilities.
+// +kubebuilder:validation:Enum=FRR-k8s
+type RoutingCapabilitiesProvider string
+
+const (
+	// RoutingCapabilitiesProviderFRRk8s determines FRR-k8s is providing
+	// advanced routing capabilities.
+	RoutingCapabilitiesProviderFRRk8s RoutingCapabilitiesProvider = "FRR-k8s"
+)
+
+// AdditionalRoutingCapabilities describes components and relevant configuration providing
+// advanced routing capabilities.
+type AdditionalRoutingCapabilities struct {
+	// providers is a set of enabled components that provide additional routing
+	// capabilities. Valid values are:
+	// - "FRR-k8s": Enables FRR routing capabilities through the deployment of
```

Comment:
This won't render nicely when it gets converted to `oc explain`. Use complete sentences rather than bullets
```suggestion
	// capabilities. Valid values are FRR-k8s.
	// FRR-k8s enables FRR routing capabilities through the deployment of
```